test/spec/app_spec_helpers.rb

```diff
@@ -8,6 +8,23 @@ def get_element(selector, type = :css)
     }
   end
 
+  def time_handle(selector, value, type = :css, visible = true)
```

Comment:
Is this method used anywhere? I didn't find usage on this diff.

Reply:
No, can be removed.